pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -361,13 +356,36 @@ func (c *Controller) Network(endpointIP string, labels labels.Instance) network.
 	return ""
 }
 
+// RegisterSelfAsHandler register controller as handler for global subscribed targets, such as MeshConfig and NetworkConfig.
+func (c *Controller) registerSelfAsHandler() {
```

Comment:
Looks not should be grouped together at all.

Reply:
deleted.